spec/features/departments_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.feature "Department" do
+  describe "user in the editor role" do
+    before do
+      somebody = create(:user)
```

Comment:
I'm going to merge this, so please don't change it now, but if you are willing to help more, @david135 has provided a helper for this logging in boilerplate.